pubsub/rabbitmq/rabbitmq.go

```diff
@@ -358,7 +358,7 @@ func (r *rabbitMQ) prepareSubscription(channel rabbitMQChannelBroker, req pubsub
 		// declare dead letter exchange
 		dlxName := fmt.Sprintf(defaultDeadLetterExchangeFormat, queueName)
 		dlqName := fmt.Sprintf(defaultDeadLetterQueueFormat, queueName)
-		err = r.ensureExchangeDeclared(channel, dlxName, fanoutExchangeKind)
+		err = r.ensureExchangeDeclared(channel, dlxName, fanoutExchangeKind, true, r.metadata.DeleteWhenUnused)
```

Comment:
why is durable always true in this case? would users of the deadletter feature expect this to be the case?

Reply:
I am following the comment https://github.com/dapr/components-contrib/pull/2836/files#diff-6d69b213107781018930a58cb30f686338377af29091816676b2e22986bb84acR369, so should I add configuration field to allow user declare this?

Reply:
Got it.